app/controllers/settings/base_controller.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class Settings::BaseController < Sellers::BaseController
+  layout "inertia"
+
+  inertia_share do
+    {
+      settings: {
+        pages: settings_presenter.pages
```

Comment:
Since you use partial reloads to refresh only `member_infos`, you can wrap this within a lambda so it is only evaluated on standard visits
```suggestion
        pages: -> { settings_presenter.pages }
```